main/com.spotify/index.js

```diff
@@ -857,6 +857,7 @@ module.exports = class SpotifyDevice extends Tp.BaseDevice {
         }
         // spotifyd active
         if (this.spotifyd) {
+            console.log('trying to run spotifyd');
```

Comment:
debugging code sneaked in?

Reply:
It's actually intentional, I want to keep this log in so it's easier to debug in the future.